powerspikegg/go_gateway/computation.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"fmt"
+	"github.com/golang/protobuf/jsonpb"
+	"golang.org/x/net/context"
+	"net/http"
+	lolpb "powerspike.gg/powerspikegg/rawdata/public/leagueoflegends_gopb"
+	computation "powerspike.gg/powerspikegg/serving/match_computation_gopb"
+	"strconv"
+	"strings"
+)
+
+func getFeature(ctx context.Context, client computation.MatchComputationClient, summonerID int32, matchID int64, region lolpb.Region) (*computation.MatchComputationFeature, error) {
+	request := &computation.MatchComputationRequest{
+		ModelName: "Not implemented yet",
+		Match: &lolpb.MatchReference{
+			Id:     float64(matchID), // TODO(archangel): this cast shouldn't be there as a match shouldn't have a floating ID
+			Region: region,
+		},
+		SummonerId: summonerID,
+	}
+
+	response, err := client.GetFeature(ctx, request)
+	return response, err
+}
+
+func parseComputationRequestParameters(r *http.Request) (int32, int64, lolpb.Region, error) {
```

Comment:
Use a structure containing all those return parameters. Easier to read, easier to understand.